src/platform-implementation-js/dom-driver/gmail/views/gmail-nav-item-view.ts

```diff
@@ -963,7 +965,7 @@ export default class GmailNavItemView {
     }
   }
 
-  private _updateSubtitle(navItemDescriptor: any) {
+  private _updateSubtitle(navItemDescriptor: NavItemDescriptor) {
```

Comment:
Seems sensible.

Reply:
`_updateSubtitle` has only one reference in `_updateValues` where the navItemDescriptor is already typed as a `NavItemDescriptor`, so, here, navItemDescriptor will always match the `NavItemDescriptor` type.
https://github.com/InboxSDK/InboxSDK/blob/c632cc01ab4a63e0e082a8cf1ac19398999d3d75/src/platform-implementation-js/dom-driver/gmail/views/gmail-nav-item-view.ts#L1026-L1039